Makefile

```diff
@@ -58,3 +58,7 @@ test: ; $(info $(M) running test cases across all services in ingest-protocols)
 .PHONY: clean
 clean: ; $(info $(M) cleaning test & pkg/cache for $(CURDIR))
 	rm -rf $(CURDIR)/coverage.*
+
+.PHONY: generate
+generate:
+        docker run --rm -v `pwd`:`pwd` -w `pwd` znly/protoc --proto_path=./protocol/signalfx/format/log --gofast_out=./protocol/signalfx/format/log protocol/signalfx/format/log/signalfx_log.proto
```

Comment:
As we discussed offline to have this documented here as well the `znly/protoc` uses an old version of the protoc compiler which generates proto2 `ProtoPackageIsVersion2`, so would be good to use a newer docker image that generates `ProtoPackageIsVersion3`.

Reply:
For this same reason we are not using znly/protoc in Otel Collector ProtoBuf code generator. We call protoc directly. Which unfortunately requires you to install protoc and Gogoproto manually, so I am also interested in seeing a newer image to avoid the manual installation (or alternatively we can automate protoc/Gogoproto tool installation).